modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/authenticator/execution/v1/CustomAuthenticatorBaseTestCase.java

```diff
@@ -0,0 +1,583 @@
+package org.wso2.identity.integration.test.rest.api.server.authenticator.execution.v1;
```

Comment:
This is an integration test for the authenticator execution flow right ? why we have this under rest.api.server package ?